tests/ci/build-pkg.yml

```diff
@@ -28,7 +28,13 @@ steps:
       echo "Expecting value for _GITHUB_API_TOKEN"
       exit 1
     fi
-    git clone --depth 1 -b dev https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools vtools
+
+    git clone --depth 1 https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools vtools
+    vtools_ref=$(cat tests/ci/VTOOLS_VERSION)
+    cd vtools
+    git fetch origin $vtools_ref
```

Comment:
i think this defeats the purpose of `--depth 1`? you can do something like this https://github.com/vectorizedio/redpanda/blob/095eaaffb696bd5d16f6b80db418418b8094007f/tests/ci/build.yaml#L18 instead to optimize the fetch.

Reply:
an alternative is to do `git fetch --depth=1 origin ...`

Reply:
Yes. My bad, both clone and fetch should be `--depth 1`